tests/test_logseq.py

```diff
@@ -30,7 +30,7 @@ class TestLogseqIssue(AbstractServiceTest, ServiceTest):
             {"id": 1777},
             {"id": 7070},
         ],
-        "content": "DOING [#A] Do something #[[Test tag]] #[[TestTag]] #TestTag",
+        "content": "DOING [#A] Do something #[[Test tag one]] #[[TestTagTwo]] #TestTagThree",
```

Comment:
Would be nice to test the bug we're fixing, e.g.:
```suggestion
        "content": "DOING [#A] Do something http://example.com/page#NotATag #[[Test tag one]] #[[TestTagTwo]] #TestTagThree",
```